tests/unit/modules/test_consul.py

```diff
@@ -0,0 +1,1829 @@
+import salt.modules.consul as consul
```

Comment:
We are currently in the process of migrating all of our tests to pytest. Can you please move these new tests to the file: tests/pytests/unit/modules/test_consul.py  and write the tests in pytest.